FE/src/pages/QuestionSearchPage/QuestionSearchPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useLocation } from 'react-router-dom';
+import { QuestionList } from '../../components';
+import { Main, Header, InnerDiv } from './QuestionSearchPage.style';
+
+const QuestionSearchPage = () => {
+  const location = useLocation();
+  const searchQuery = new URLSearchParams(location.search).get('query');
+  alert(searchQuery);
+
+  return (
+    <Main>
+      <InnerDiv className="inner">
+        <Header>검색 결과</Header>
+        <QuestionList isSearching={true} />
```

Comment:
🚨여기서 빌드 에러 나네요.

QuestionList 컴포넌트의 props가 questionListData로 수정되었는데,
수정된 props가 반영되지 않아 생기는 에러 같습니다. 수정 부탁드려요.

Reply:
바로 다음 PR에서 API로직이 반영될테니, 이번 PR에서는 컴포넌트 랜더링 자체를 막아놓는 것도 괜찮을 것 같아요.
더미 데이터를 만드는 건 너무 시간이 많이 소요될 것 같네요..
```suggestion
        {/* <QuestionList isSearching={true} /> */}
```

Reply:
또 같은 문제가 발생한 것 같은데 저는 이미 QuestionSearchPage를 수정한 상태지만 수정 내용은 다음 pr에 반영하는 게 맞다고 생각해서 push는 하지 않았고, 그러다 보니 제가 build 할 때는 이상이 생기지 않았던 것 같네요.
아래에 방식으로 막아 놓는 게 괜찮을 것 같네요.